include/javabind/binding.hpp

```diff
@@ -377,6 +377,7 @@ namespace javabind
     };
 
     struct EnumBinding {
+        std::vector<std::function<void(const std::unordered_map<std::string, EnumValue>&)>> initializers;
         std::vector<std::string_view> names;
```

Comment:
Do we want to keep these variables on the public interface of `EnumBinding`, or hide these behind member functions? We already have `add` and `initialize`. It might make sense to add other member functions to extend `initializers`.